app/models/application_record.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationRecord < ActiveRecord::Base
+  include ActionView::Helpers::TranslationHelper
```

Comment:
Why do we need this?

Reply:
Added if wanted inside Model, or we can add later if needed.

Reply:
I think t alias will work without this.